renovate.json

```diff
@@ -9,7 +9,12 @@
 		{
 			"extends": "monorepo:wordpress",
 			"separateMajorMinor": false,
-			"prPriority": 2
+			"prPriority": 2,
+			"assignees": [
+				"@Automattic/cylon",
+				"@Automattic/ganon",
+				"@Automattic/luna",
```

Comment:
I'm not sure trailing commas are legal in JSON...

Reply:
Thanks! Cleaned those out.